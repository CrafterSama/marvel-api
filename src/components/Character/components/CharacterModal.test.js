import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';
import Charactermodal from './Charactermodal';

configure({ adapter: new Adapter() });

describe('Test for <Charactermodal /> Component', () => {

  test('matches snapshot', () => {
    const wrapper = shallow(
      <Charactermodal
        isOpen={false}
        toggle={() => {}}
        data={{ data: {} }}
      />
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
