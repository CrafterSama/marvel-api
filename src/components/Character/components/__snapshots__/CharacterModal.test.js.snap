// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <Charactermodal /> Component matches snapshot 1`] = `
<Modal
  isOpen={false}
  toggle={[Function]}
>
  <CharacterComicsInfo
    data={
      Object {
        "data": Object {},
      }
    }
    toggle={[Function]}
  />
</Modal>
`;
